CHANGELOG-1.x.md

```diff
@@ -1,3 +1,5 @@
+- [HASH HERE](URL HERE) feat: Pfe-Primary-Detail - Added much improved a11y and keyboard navigation, added mobile optimized layout behavior, and added thorough tests using new framework.
```

Comment:
```suggestion
# 1.11.0 (2021)

- [](https://github.com/patternfly/patternfly-elements/commit/) feat: Pfe-Primary-Detail - Added much improved a11y and keyboard navigation, added mobile optimized layout behavior, and added thorough tests using new framework.

```

Reply:
We're still looking at how to automate this so hopefully we won't be manually doing this much longer.  This format is easier for the person doing the releases to update.